appConfig/appConfig.snd4.yaml

```diff
@@ -13,8 +13,8 @@
 - key: NOTIFY_API_KEY
   value: "{{servicename}}-NOTIFY-API-KEY"
   type: "keyvault"
-- key: NOTIFY_TEMPLATE_ID
-  value: "{{servicename}}-NOTIFY-TEMPLATE-ID"
+- key: NOTIFY_MESSAGES_TEMPLATE_ID
+  value: "{{servicename}}-NOTIFY-MESSAGES-TEMPLATE-ID"
```

Comment:
This has also been updated in the [ADO library for SND4](https://dev.azure.com/defragovuk/DEFRA-FFC/_library?itemType=VariableGroups&view=VariableGroupView&variableGroupId=1950&path=ffc-sfd-messages-processor-snd4).